pkg/tfbridge/schema.go

```diff
@@ -956,7 +980,7 @@ func MakeTerraformOutputs(p shim.Provider, outs map[string]interface{},
 
 		// Next perform a translation of the value accordingly.
 		out := MakeTerraformOutput(p, value, tfi, psi, assets, rawNames, supportsSecrets)
-		//if !out.IsNull() {
+		// if !out.IsNull() {
 		result[name] = out
```

Comment:
If we're going to touch this, we may as well fix it.
```suggestion
		result[name] = MakeTerraformOutput(p, value, tfi, psi, assets, rawNames, supportsSecrets)
```